<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Seccion_LLaves_de_Puertas</class>
 <widget class="QWidget" name="Seccion_LLaves_de_Puertas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>665</width>
    <height>466</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>46</red>
        <green>47</green>
        <blue>48</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>46</red>
        <green>47</green>
        <blue>48</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QWidget" name="w_seccion_2" native="true">
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(77, 77, 77);</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_53">
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_54">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_28">
          <property name="spacing">
           <number>10</number>
          </property>
          <item>
           <widget class="QWidget" name="widget_llaves_de_puertas" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_55">
             <property name="topMargin">
              <number>20</number>
             </property>
             <property name="bottomMargin">
              <number>20</number>
             </property>
             <item>
              <widget class="QCheckBox" name="rb_siempre_abierto">
               <property name="font">
                <font>
                 <family>Segoe UI</family>
                 <pointsize>10</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Siempre Abierto</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox">
               <property name="title">
                <string/>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout">
                <property name="spacing">
                 <number>10</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QRadioButton" name="rb_llaves_de_exterior">
                  <property name="font">
                   <font>
                    <family>Segoe UI</family>
                    <pointsize>10</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>Llaves de exterior</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="rb_llaves_maestras">
                  <property name="font">
                   <font>
                    <family>Segoe UI</family>
                    <pointsize>10</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>Llaves maestras</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="rb_llave_especifica">
                  <property name="font">
                   <font>
                    <family>Segoe UI</family>
                    <pointsize>10</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>Llave espec√≠fica</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="rb_sin_comprobar">
                  <property name="font">
                   <font>
                    <family>Segoe UI</family>
                    <pointsize>10</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>Sin comprobar</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_62">
            <property name="topMargin">
             <number>80</number>
            </property>
            <property name="bottomMargin">
             <number>80</number>
            </property>
            <item>
             <widget class="QLabel" name="lb_llaves_de_exterior">
              <property name="font">
               <font>
                <family>Segoe UI</family>
                <pointsize>10</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(54, 141, 206);</string>
              </property>
              <property name="text">
               <string>Llaves de Exterior</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="MyLabelSpinner" name="l_llaves_de_exterior">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>22</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>24</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Segoe UI</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: #EFEFEF; 
border-radius: 6px;
color:rgb(50, 50, 50);</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_63">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="topMargin">
             <number>80</number>
            </property>
            <property name="bottomMargin">
             <number>80</number>
            </property>
            <item>
             <widget class="QLabel" name="lb_llaves_maestras">
              <property name="font">
               <font>
                <family>Segoe UI</family>
                <pointsize>10</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(54, 141, 206);</string>
              </property>
              <property name="text">
               <string>Llaves Maestras</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="MyLabelSpinner" name="l_llaves_maestras">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>22</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>24</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Segoe UI</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: #EFEFEF; 
border-radius: 6px;
color:rgb(50, 50, 50);</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QWidget" name="widget_llaves_especificas" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_66">
             <property name="spacing">
              <number>6</number>
             </property>
             <property name="bottomMargin">
              <number>20</number>
             </property>
             <item>
              <widget class="QLabel" name="label_31">
               <property name="font">
                <font>
                 <family>Segoe UI</family>
                 <pointsize>10</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(54, 141, 206);</string>
               </property>
               <property name="text">
                <string>Llave Espec√≠fica</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="cb_Tienen_todos_los_vecinos">
               <property name="font">
                <font>
                 <family>Segoe UI</family>
                 <pointsize>10</pointsize>
                 <weight>50</weight>
                 <bold>false</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>Tienen todos los vecinos</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="cb_Tiene_algun_vecino">
               <property name="font">
                <font>
                 <family>Segoe UI</family>
                 <pointsize>10</pointsize>
                 <weight>50</weight>
                 <bold>false</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>Tiene algun vecino</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="cb_Presidente">
               <property name="font">
                <font>
                 <family>Segoe UI</family>
                 <pointsize>10</pointsize>
                 <weight>50</weight>
                 <bold>false</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>Presidentes</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="cb_Administracion">
               <property name="font">
                <font>
                 <family>Segoe UI</family>
                 <pointsize>10</pointsize>
                 <weight>50</weight>
                 <bold>false</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>Administracion</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="cb_Conserje">
               <property name="font">
                <font>
                 <family>Segoe UI</family>
                 <pointsize>10</pointsize>
                 <weight>50</weight>
                 <bold>false</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>Conserje</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="cb_Copia_en_gestor_aguas">
               <property name="font">
                <font>
                 <family>Segoe UI</family>
                 <pointsize>10</pointsize>
                 <weight>50</weight>
                 <bold>false</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>Copia en gestor aguas</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_65">
          <property name="bottomMargin">
           <number>20</number>
          </property>
          <item>
           <widget class="MyLabelSpinner" name="l_utilidad_de_llave_maestra">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>22</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Segoe UI</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: #EFEFEF; 
border-radius: 6px;
color:rgb(50, 50, 50);</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_57">
          <item>
           <widget class="QLabel" name="label_11">
            <property name="font">
             <font>
              <family>Segoe UI</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(54, 141, 206);</string>
            </property>
            <property name="text">
             <string>Nota</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="MyPlainTextShine" name="pt_nota">
            <property name="font">
             <font>
              <family>Segoe UI</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: #EFEFEF; 
border-radius: 8px;
color:rgb(50, 50, 50);</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="topMargin">
             <number>20</number>
            </property>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pb_aceptar">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>0</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Segoe UI</family>
                <pointsize>10</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">border-radius: 10px;
color:rgb(120, 120, 120);</string>
              </property>
              <property name="text">
               <string>ACEPTAR</string>
              </property>
              <property name="icon">
               <iconset resource="assets.qrc">
                <normaloff>:/icons/accept.png</normaloff>:/icons/accept.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pb_cancelar">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>0</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Segoe UI</family>
                <pointsize>10</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">border-radius: 10px;
color:rgb(120, 120, 120);</string>
              </property>
              <property name="text">
               <string>CANCELAR</string>
              </property>
              <property name="icon">
               <iconset resource="assets.qrc">
                <normaloff>:/icons/cancel.png</normaloff>:/icons/cancel.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyLabelSpinner</class>
   <extends>QLabel</extends>
   <header location="global">mylabelspinner.h</header>
  </customwidget>
  <customwidget>
   <class>MyPlainTextShine</class>
   <extends>QPlainTextEdit</extends>
   <header location="global">myplaintextshine.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="assets.qrc"/>
 </resources>
 <connections/>
</ui>
